packages/lesswrong/server/vulcan-lib/utils.ts

```diff
@@ -15,12 +16,18 @@ export const sanitize = function(s) {
     allowedAttributes:  {
       ...sanitizeHtml.defaults.allowedAttributes,
       img: [ 'src' , 'srcset'],
-      figure: ['style']
+      figure: ['style'],
+      td: ['rowspan', 'colspan'],
+      span: ['style']
     },
     allowedStyles: {
       ...sanitizeHtml.defaults.allowedStyles,
       'figure': {
         'width': [/^(?:\d|\.)+(?:px|em|%)$/]
+      },
+      span: {
+        // From: https://gist.github.com/olmokramer/82ccce673f86db7cda5e#gistcomment-3119899
+        color: [/(#([\da-f]{3}){1,2}|(rgb|hsl)a\((\d{1,3}%?,\s?){3}(1|0?\.\d+)\)|(rgb|hsl)\(\d{1,3}%?(,\s?\d{1,3}%?){2}\))/]
```

Comment:
This regex accepts colors like "#123456" and "rgba(0,0,0,0.5)" but not color-names like "blue". This version adds color words:
```suggestion
        color: [/([a-z]+|#([\da-f]{3}){1,2}|(rgb|hsl)a\((\d{1,3}%?,\s?){3}(1|0?\.\d+)\)|(rgb|hsl)\(\d{1,3}%?(,\s?\d{1,3}%?){2}\))/]
```